app/templates/components/permission-language.hbs

```diff
@@ -0,0 +1,18 @@
+{{yield}}
+{{#if (eq model.name "engrXiv")}}
+    <div class="text-center permission-footer-language ">
+        <span>arXiv is a trademark of Cornell University, used under license. This license should not be understood to indicate endorsement of content on engrXiv by Cornell University or arXiv.</span>
```

Comment:
We are using i18n to enable us to do localization in the future, which means that we keep language in a separate file. This will let us translate into different languages easily. I don't know the exact method for dealing with this, but you'll want to put these strings into https://github.com/CenterForOpenScience/ember-preprints/blob/develop/app/locales/en/translations.js and pull from there.